apps/iiif/manifests/views.py

```diff
@@ -63,6 +63,38 @@ def get(self, request, *args, **kwargs): # pylint: disable = unused-argument
                 safe=False
             )
 
+class AllVolumesCollection(View):
+    """Endpoint for all volumes manifest."""
+    def get_queryset(self):
+        """Get all manifest objects.
+
+        :return: All manifest objects
+        :rtype: django.db.models.QuerySet
+        """
+        return Manifest.objects.all()
+
+    def get(self, request, *args, **kwargs): # pylint: disable = unused-argument
+        """Responds to HTTP GET request for all manifests.
+
+        :return: IIIF representation of all volumes manifest
```

Comment:
Same here, "return: IIIF representation of all volumes collection" in this docstring

Reply:
Updated as suggested